src/main/scala/dal/SqlestDb.scala

```diff
@@ -0,0 +1,23 @@
+package dal
+
+import sqlest._
+import com.typesafe.config._
+//import server.Config._
```

Comment:
This is meant to be fetching the data out of the application.conf text file. We need this for the url, username and password of the database. I will provide some extra info about this to Sean,